modules/task/execution.go

```diff
@@ -133,6 +133,15 @@ func Register(fn interface{}) error {
 	return nil
 }
 
+// MustRegister registers a function for an async task or panics
+// Since Task registration is performed in main() we want to fail-fast and
+// reduce the amount of error-checking
+func MustRegister(fn interface{}) {
```

Comment:
Actually, just read the code. It defaults to a noop backend. If the programmer registers a nil backend somehow, that means they created a backend provider that returns a nil interface **and** a nil error, which should not be the case (and is not the case in our internal implementation). So I'm going to skip the TODO for now.